media/css/newsletter/ios.less

```diff
@@ -0,0 +1,154 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+.feature {
+    margin-bottom: @baseLine;
+    text-align: center;
+}
+
+#newsletter-form {
+    .span(8);
```

Comment:
You can do <code>width: (@gridColumnWidth * 8) + (@gridGutterWidth * 7);</code> to get the width of 8 columns and you won't have to override the extra stuff (float and margins) this mixin generates.